apps/smartling/lambda/src/index.ts

```diff
@@ -17,6 +17,7 @@ function requestProxier(proxyUrl: string, proxyPathPrefix = '') {
 export function makeApp(fetchFn: any, issuer: any) {
   const app = express() as Express;
 
+  // tslint:disable-next-line
```

Comment:
Would be great to know why these are now needed. Feel free to add a comment

Reply:
I fixed the issue tslint was nagging about. 
The rest of this file could need some TLC eventually and we should consider replacing TSLint with ESLint.